tests/test_erupt.py

```diff
@@ -1,24 +1,25 @@
 from typing import List
 
 import pytest
+import os
+import sys
 
 import pandas as pd
 import numpy as np
 from sklearn.dummy import DummyClassifier
 
-from auto_causality.erupt import ERUPT
+root_path = root_path = os.path.realpath("..")
+sys.path.append(os.path.join(root_path, "dowhy"))
```

Comment:
We should never have to do this and I don't see a `dowhy` directory in the project. What's this for?

Reply:
Hi @markharley  thanks for your comment! Egor asked us to download the most recent version of dowhy straight from Github has it includes some features he proposed which haven't made it into the official release yet. I'm happy to remove this line

Reply:
Ah okay. We can install directly from GitHub by adding this to requirements.txt,
```
git+https://github.com/microsoft/dowhy@52dd55f#egg=dowhy
```
`52dd55f` is the current latest commit on their master branch (it's a good idea to pin this so that we can ensure we're all using the same version). Specifying `egg=dowhy` picks the `dowhy` package out of the repo to install.

Reply:
thanks, I've addressed these points in  https://github.com/transferwise/auto-causality/pull/14/commits/d877c077baa1e6a1e9529db3b07130cabcc1f60b

Reply:
Hi @markharley can this PR be merged into main? The only test that is failing is Egor's end-to-end test: of routines unrelated to this PR:
```
FAILED tests/test_full_flow.py::TestFullRun::test_full_run - AttributeError: ...
================== 1 failed, 16 passed, 42 warnings in 15.52s ==================
```
It's happening due to the update to flaml 0.9.5 and could be fixed by replacing his AutoMLWrapper with direct calls to AutoML in the script. The linter fails due to the issue addressed in PR #20